sphinx/writers/latex.py

```diff
@@ -64,6 +64,23 @@
                                        'lowerroman': r'\roman',
                                        'upperroman': r'\Roman',
                                    })  # type: Dict[unicode, unicode]
+PDFLATEX_DEFAULT_FONT_PKG = r'''
+\usepackage{times}
+\expandafter\ifx\csname T@LGR\endcsname\relax
+\else
+% LGR was declared as font encoding
+  \substitutefont{LGR}{\rmdefault}{artemisia}   % gfsartemisia
+  \substitutefont{LGR}{\sfdefault}{neohellenic} % gfsneohellenic
+  \substitutefont{LGR}{\ttdefault}{cmtt}        % cbfonts
+\fi
+\expandafter\ifx\csname T@T2A\endcsname\relax
+\else
+% T2A was declared as font encoding
+  \substitutefont{T2A}{\rmdefault}{fcm}
+  \substitutefont{T2A}{\sfdefault}{fcs}
+  \substitutefont{T2A}{\ttdefault}{fct}
```

Comment:
this would be bad if T2A is main font encoding; but this can only happen if user is building a project in a Cyrillic language, and then certainly ``'fontpkg'`` key is used, because "times" package has no Cyrillic support with pdflatex. Then this default is overwrittent by user setting hence these font substitution are not executed. This is why I did not add a test to check if T2A is the main font encoding (`\LastDeclaredEncoding` could have been used in a latex conditional)